common/components/forms/ContactForm.jsx

```diff
@@ -0,0 +1,165 @@
+// @flow
+
+import React from 'react';
+import ProjectAPIUtils from '../utils/ProjectAPIUtils.js';
+import ReCAPTCHA from "react-google-recaptcha";
+import Guard from '../utils/guard.js'
+import apiHelper from '../utils/api.js'
+
+type State = FormFields & ControlVariables
+
+type FormFields = {|
+  fname: string,
+  lname: string,
+  emailaddr: string,
+  message: string
+|};
+
+type ControlVariables = {|
+  sendStatusMessage: string,
+  sendStatusClass: string,
+  reCaptchaValue: string,
+|};
+
+class ContactForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fname: '',
+      lname: '',
+      emailaddr: '',
+      message: '',
+      sendStatusMessage: null,
+      sendStatusClass: null,
+      reCaptchaValue: null,
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = Guard.duplicateInput(this.handleSubmit.bind(this), 1000, {'leading': true});
```

Comment:
```suggestion
    this.handleSubmit = Guard.duplicateInput(this.handleSubmit.bind(this));
```
No need to provide arguments when they match the default.

Reply:
Makes sense, updated.